libsol/transaction_printers.c

```diff
@@ -532,6 +536,33 @@ static int print_spl_associated_token_account_create_with_transfer(const PrintCo
     return 0;
 }
 
+static int print_compute_budget_unit_price(InstructionInfo* info) {
+    SummaryItemPayload* item_payload = transaction_summary_payload_priority_fees_item();
+    summary_item_payload_set_u64(item_payload, info->compute_budget.change_unit_price.units);
+    return 0;
+}
+
+static int print_compute_budget_unit_limit(InstructionInfo* info){
+    SummaryItemPayload* item_payload = transaction_summary_payload_compute_units_limit_item();
+    summary_item_payload_set_u64(item_payload, info->compute_budget.change_unit_limit.units);
+    return 0;
+}
+
+static int print_compute_budget(InstructionInfo* info){
+    switch (info->compute_budget.kind) {
+        case ComputeBudgetChangeUnitLimit:
+            print_compute_budget_unit_limit(info);
+            break;
+        case ComputeBudgetChangeUnitPrice:
+            print_compute_budget_unit_price(info);
+            break;
+        case ComputeBudgetRequestUnits:
+        case ComputeBudgetRequestHeapFrame:
+            break;
+    }
+    return 0;
+}
+
```

Comment:
these helpers belong in `compute_budget_instruction.c`. the ones in this file declare entire transaction constructions, not individual instruction handling